pinot-core/src/main/java/com/linkedin/pinot/core/segment/creator/impl/SegmentColumnarIndexCreator.java

```diff
@@ -414,6 +420,10 @@ public static void removeColumnMetadataInfo(PropertiesConfiguration properties,
     properties.clearProperty(getKeyFor(column, TOTAL_NUMBER_OF_ENTRIES));
     properties.clearProperty(getKeyFor(column, IS_AUTO_GENERATED));
     properties.clearProperty(getKeyFor(column, DEFAULT_NULL_VALUE));
+    properties.clearProperty(getKeyFor(column, DERIVED_METRIC_TYPE));
```

Comment:
Seems these two are independent bug you found, and not related to this PR, right?